### DISCUSSION TITLE
SolarMoon Incorrect Logo

### DISCUSSION THREAD

**Discussion by Nickishweb3** [phantom/docs#147] [Help & Support] [answered]
### Summary

SolarMoon logo is showing is as our old logo what the best way to update 

https://birdeye.so/token/2kMpEJCZL8vEDZe7YPLMCS9Y3WKSAMedXBn7xHPvsWvi?chain=solana

### Steps to Reproduce

Open phantom and view SolarMoon tokens in Wallet

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @Nickishweb3,

Would you mind pointing to the correct logo? Your metadata uri gives the following text: ```Starting August 1st, 2023 API Keys are required to use thirdweb services. Learn more: https://blog.thirdweb.com/changelog/api-keys-to-access-thirdweb-infra```

**Comment by Nickishweb3** [comment]
Hi Adam,

Sure!

https://pbs.twimg.com/profile_images/1702167599134453760/sDbTwMAP_400x400.jpg

Here is the correct logo currently on our Twitter

Kind Regards
Nick
[https://pbs.twimg.com/profile_images/1702167599134453760/sDbTwMAP_400x400.jpg]

________________________________

**Reply by adamdelphantom** [reply, marked as answer]
@Nickishweb3 you will need to update your token metadata to reflect the correct token logo.
